sumologic/resource_sumologic_connection.go

```diff
@@ -85,6 +85,12 @@ func resourceSumologicConnection() *schema.Resource {
 				Optional:     true,
 				ValidateFunc: validation.StringInSlice([]string{"Incident", "Event"}, false),
 			},
+			"resolution_payload": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: validation.StringIsJSON,
+				Computed:     true,
```

Comment:
Added computed:true for resolution payload because, if nothing is set by user, a default value is set on server side and returned.

Reply:
If user specified value is different from the server-side value during the update, does `terraform apply` apply the new change for this field?

Reply:
Yes. it will apply that value